src/User/Event/FormEvent.php

```diff
@@ -14,6 +14,12 @@
 use yii\base\Event;
 use yii\base\Model;
 
+/**
```

Comment:
Please, remove all classes phpDocs to match the styling guide throughout the library.

Reply:
how a PHPDoc style for a class? This is template from PHPStorm...

Reply:
And... This mark a declaration for a magic property.

```php
 * @property Model $form
```

Reply:
I don't use any phpDoc template for class. If you guys think is needed then should be on the entire library and in regards to @package 2amigos/yii-usuario

Reply:
> and in regards to @Package 2amigos/yii-usuario

In yii2 framework core code last lines is a author list and version when included class... https://github.com/yiisoft/yii2/blob/6960d2cfe670f002a5611f68a948daa627460dcd/framework/db/ActiveRecord.php#L73-L79

Reply:
So... This is correct? 

```php
/**
 * Class FormEvent
 * @package Da\User\Event
 * @Package 2amigos/yii-usuario
 * 
 * @property Model $form
 */
```

Reply:
Tell about your version. Example?

Reply:
My version has no phpCode for classes unless necessary (for magic methods and/or properties)

Reply:
w/o  `@property Model $form` for magic methods we can't use autocomplete in IDE. And we get warning for this.
![peek 2018-10-15 16-31](https://user-images.githubusercontent.com/5769211/46954080-538a9e80-d098-11e8-9248-39402472ad79.gif)

Autocomplete can use data from getter, but not use magic.

Reply:
I would simply add this at the top of the class: 

````
/**
 * @property Model $form
 */
```
If @maxxer also things this is a requirement, I'll update all classes with appropriate phpDocs. Tried to avoid that in the past for the sake of speed.